README.md

```diff
@@ -34,14 +34,25 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+I would say semantic HTML makes it much easier to see what part of the code is each text.
```

Comment:
Yes, it allows you and other developers to clearly understand code. It also allows for better indexing such as SEO.